docs/src/pages/components/portal/portal.md

```diff
@@ -23,3 +23,11 @@ The component is used internally by the [`Modal`](/components/modal/) and [`Popp
 
 React [doesn't support](https://github.com/facebook/react/issues/13097) the [`createPortal()`](https://reactjs.org/docs/portals.html) API on the server.
 You have to wait for the client-side hydration to see the children.
+
+## Unstyled
+
+As a component that does not have any styles, the portal also comes with the unstyled package.
```

Comment:
I like this. What do you think about using this exact wording for https://next--material-ui.netlify.app/components/trap-focus/#unstyled?

```suggestion
As the component that does not have any styles, it also comes with the unstyled package.
```

Reply:
Maybe
```suggestion
As the component does not have any styles, the portal also comes with the unstyled package.
```

?

Reply:
It looks the same :)